modules/nf-core/jvarkit/sam2tsv/main.nf

```diff
@@ -0,0 +1,52 @@
+process JVARKIT_SAM2TSV {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/jvarkit:2024.08.25--hdfd78af_1':
+        'biocontainers/jvarkit:2024.08.25--hdfd78af_1' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    tuple val(meta2), path(fasta), path(fasta_index), path(fasta_dict)
+    tuple val(meta3), path(regions_file)
```

Comment:
I think this regions_file should probably be attached to the input, as it may well be sample dependent.
(I used this exact subtool in a local module in an in-house pipeline a few months ago, where I had different regions I wanted to extract per sample).

Reply:
Good idea, this makes sense. I have added the regions file to the sample input. Ran prettier and validated the yml files now. Hope everything is fine now :)